client/Android/FreeRDPCore/res/values-zh/strings.xml

```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<resources>
+    <!-- Button labels -->
+    <string name="yes">是</string>
+    <string name="no">否</string>
+    <string name="cancel">取消</string>
+    <string name="cont">继续</string>
+    <string name="login">登陆</string>
```

Comment:
And, i don't have any knowledge of photoshop, so i can't translate the word in the picture now. But i will try to learn some skill of photoshop to translate it. Thank you for your reply.